packages/commonwealth/client/scripts/views/pages/CreateCommunity/steps/CommunityTypeStep/helpers.ts

```diff
@@ -2,16 +2,23 @@ import { ChainBase } from '@hicommonwealth/shared';
 import { CommunityType } from 'views/components/component_kit/new_designs/CWCommunitySelector';
 
 export const communityTypeOptions = [
+  {
+    type: CommunityType.Base,
+    chainBase: ChainBase.Base,
+    title: 'BASE',
+    isRecommended: true,
+    description:
+      'Base in an Ethereum layer 2 network with high TVL and low transaction fees',
+  },
```

Comment:
please update the code in `CommunityTypeStep.tsx` as currently it is messed up and this is coincidence that it is rendered in the good shape. Talking here about destructuring at the very top of the image. After your change, first option is base, second blast, third ethereum 

![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/780cd3de-e8e4-48d2-a070-edae848e399e)